src/app/free-navigation-mode/free-navigation-mode.component.scss

```diff
@@ -6,7 +6,12 @@
 }
 
 // Adjust the styling of wizard steps so that optional steps look like default ones.
+// (in preudo-code, $wz-colors['_']['optional'] := $wz-colors['_']['default'])
+$wz-colors: map-merge($wz-colors, (
+  '_': map-merge(map-get($wz-colors, '_'), (
+    'optional': map-get(map-get($wz-colors, '_'), 'default')
```

Comment:
Is it also possible to only override certain parts of `optional` without overriding all its fields? For example if we only want to change the border color but nothing else. I think we should add such a scenario.

Reply:
> Is it also possible to only override certain parts of optional without overriding all its fields? For example if we only want to change the border color but nothing else. 

Yes, it is possible.

But demonstrating it in the demo application will be awkward. I tried to update the 'Custom global CSS' demo using SCSS, but there are some things which cannot be defined via SCSS variables, so you end up with a mixture of SCSS and CSS overrides in one file, which is harder to read and maintain in comparison with the CSS-only approach. For things which the 'Custom global CSS' demo does, I myself would rather prefer CSS overrides than SCSS.

Anyway, I am going to commit a new 'Custom global SCSS' demo. Please take a look and decide whether it is useful to have or not. In my opinion, it may confuse viewers because the code is more complicated than necessary (that is, than in the CSS-only approach).